tests/unit/data_providers/test_new_tab_slate_provider.py

```diff
@@ -100,20 +102,49 @@ async def test_rank_by_thompson_sampling(
             top_ranked_ids.append(ranked_items[0].id)
             bottom_ranked_ids.append(ranked_items[-1].id)
 
-            for item in ranked_items:
-                if item.id == fixture_item_id_with_high_ctr:
-                    # corpus_engagement.json fixture has UPDATED_AT set to '2023-05-01T12:00:00Z'
-                    assert item.ranked_with_engagement_updated_at == datetime(2023, 5, 1, 12, 0, 0, tzinfo=timezone.utc)
-                else:
-                    # other items do not exist in corpus_engagement.json fixture
-                    assert item.ranked_with_engagement_updated_at is None
```

Comment:
I was confused why I added a assertion on `UPDATED_AT` to this test. It can easily break when additional fixture data is added, and it is not necessary to test that Thompson sampling is working as expected.